src/mqueryfront/src/queryUtils.js

```diff
@@ -0,0 +1,38 @@
+export const isStatusFinished = (status) =>
+    ["done", "cancelled", "failed", "expired"].includes(status);
+
+export const getClassForStatus = (status) => {
+    let classSufix = null;
+
+    switch (status) {
+        case "done":
+            classSufix = "success";
+            break;
+        case "processing":
+        case "querying":
+            classSufix = "info";
+            break;
+        case "expired":
+            classSufix = "warning";
+            break;
+        case "cancelled":
+            classSufix = "danger";
+            break;
```

Comment:
We have a new status now - `removed. Maybe use `dark` for it?

Reply:
"dark" seems reasonable. Works for background and badge (`bg-dark,` `badge-dark`).